package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "version": "0.0.33",
+  "version": "0.0.34",
   "versions": [
-    "10.1.0",
+    "12.0.0-beta.0",
```

Comment:
Can you revert the ordering so it's increasing in version numbers from the top?

Reply:
Done.